src/common/components/RichTextEditor/index.js

```diff
@@ -515,8 +513,8 @@ const RichTextEditor = props => {
           style={
             addContainer
               ? {
-                  paddingLeft: theme.spacing(2),
-                  paddingRight: theme.spacing(2),
+                  paddingLeft: 2,
```

Comment:
Issue: You can only use the mui spacing values directly inside mui components, this is not an MUI component, this should stay the same